vstgui/lib/vstguifwd.h

```diff
@@ -74,6 +74,17 @@ enum DragResult {
 	kDragError = -1
 };
 
+//----------------------------
+// @brief Bitmap Interpolation
+//----------------------------
+enum BitmapInterpolationQuality
```

Comment:
change this to : enum class BitmapInterpolationQuality
and change the members to kDefault,kLow,kMedium,kHigh.

Reply:
Okay, I tried to stick to the other enums in vstguifwd.h which aren't enum classes, either. Any specific reason why that is?

Reply:
I don't want to break old code if it is not necessary.

Reply:
Okay, thanks, that makes sense.

Reply:
I know this is off-topic, but is there any specific reason that CFrame has been marked final? Performance?

I have an instance where I need to override some functions in CFrame in order to access its pixel contents using the COffscreenContext/CBitmap approach described above. This is done to implement an eyedropper. What would be the suggested workaround here?

Reply:
The workaround would be to place everything into a root CViewContainer that you override for this case.

Reply:
That makes perfect sense. Thank You!